edx_proctoring/tests/test_views.py

```diff
@@ -2,14 +2,18 @@
 """
 All tests for the proctored_exams.py
 """
+
+from __future__ import absolute_import
+
+from datetime import datetime, timedelta
 import json
-import pytz
+from string import Template  # pylint: disable=deprecated-module
```

Comment:
Good call. I've converted this weird template logic to instead use a parameterized function.